first_lessons_dushanbe/array_sorting_bubbles.py

```diff
@@ -6,14 +6,14 @@ def list_sort(a):
     if a is None:
         return a
     count = len(a)-1
-    while count > 1:
+    while count >= 1:
```

Comment:
you change condition, it means you did find a bug in your code. Could you please add test case for this bug?